packages/api-client/src/asset/AssetAPI.ts

```diff
@@ -154,6 +154,7 @@ export class AssetAPI {
     token?: string | null,
     forceCaching: boolean = false,
     progressCallback?: ProgressCallback,
+    customUrl?: string,
```

Comment:
It feels `customUrl` might be a bit too broad especially since the function is running validation against the parameter `assetId` but might use a completely different one inside the `customUrl`. Also it's not guaranteed that the passed `customUrl` is a v3 URL. Maybe something like `urlPrefix` is more appropriate.

Reply:
Maybe even `isBotAsset`?